tests/test_dia_calculation_plugins.py

```diff
@@ -53,39 +56,149 @@ def testCalculate(self):
         # Test expected means.
         diaObject = dict()
         diaSources = pd.DataFrame(data={"ra": np.linspace(-1, 1, n_sources),
-                                        "decl": np.zeros(n_sources)})
+                                        "decl": np.zeros(n_sources),
+                                        "midPointTai": np.linspace(0,
+                                                                   n_sources,
+                                                                   n_sources)})
         plug = MeanDiaPosition(MeanDiaPositionConfig(),
                                "ap_meanPosition",
                                None)
         plug.calculate(diaObject, diaSources)
 
         self.assertAlmostEqual(diaObject["ra"], 0.0)
         self.assertAlmostEqual(diaObject["decl"], 0.0)
+        self.assertEqual(diaObject["radecTai"], 10)
 
         diaObject = dict()
         diaSources = pd.DataFrame(data={"ra": np.zeros(n_sources),
-                                        "decl": np.linspace(-1, 1, n_sources)})
+                                        "decl": np.linspace(-1, 1, n_sources),
+                                        "midPointTai": np.linspace(0,
+                                                                   n_sources,
+                                                                   n_sources)})
```

Comment:
I'm not sure if these are reasonable timestamps or not, are the units in MJD? Starting at 2450000-whatever might more accurately approximate reality.

Reply:
It doesn't matter what the values are. Only that they are round tripped.

Reply:
I guess they won't ever be anywhere near too big to store/retrieve properly, so OK.